apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,243 @@
+              {`Choose a layout for "`}
+              {props.pageName}
+              {`"`}
+            </Text>
+            <Button size="xs">
+              Start with this layout
+              <BxRightArrowAlt fontSize="1.5rem" />
+            </Button>
+          </HStack>
+        </VStack>
+        <Grid
+          w="100vw"
+          h="calc(100% - 7rem)"
+          templateColumns="repeat(4, 1fr)"
+          gap={0}
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((e) => {
+                return (
+                  <VStack
+                    key={e.layoutTypename}
+                    p="0"
+                    gap="0"
+                    cursor="pointer"
+                    onClick={() => {
+                      setSelectedLayout(e)
+                    }}
+                    _hover={
+                      selectedLayout.layoutTypename === e.layoutTypename
+                        ? {}
+                        : {
+                            '.layout-title': { color: 'base.content.brand' },
+                            '.hover-text': {
+                              opacity: 1,
+                            },
+                            '.hover-border': {
+                              borderColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.hover',
+                              ),
+                            },
+                          }
+                    }
+                    sx={
+                      selectedLayout.layoutTypename === e.layoutTypename
+                        ? {}
+                        : {
+                            '.layout-title': {
+                              transition: 'color 300ms ease-out',
+                            },
+                            '.hover-text': {
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            },
+                          }
+                    }
+                  >
+                    <Box
+                      className="hover-border"
+                      borderRadius="8px"
+                      border="2px"
+                      p="1.5rem 1.5rem 0 1.5rem"
+                      mb="1.25rem"
+                      position="relative"
+                      borderColor={
+                        selectedLayout.layoutTypename === e.layoutTypename
+                          ? 'base.divider.brand'
+                          : 'base.divider.medium'
+                      }
+                    >
+                      <Image
+                        src={e.imageSource}
+                        borderRadius="4px 4px 0px 0px"
+                        boxShadow="0px 0px 20px 0px rgba(104, 104, 104, 0.30)"
```

Comment:
nit: fien with this `rgba(xxx)` since we're rolling this out but if we have a token, please od use the token!